lib/utils/index.js

```diff
@@ -1024,6 +1024,18 @@ function download_zip_url(options = {}) {
   }));
 }
 
+/**
```

Comment:
Didn't we drop the need of `lib-es5`?
@patrick-tolosa

Reply:
not merged yet